### ISSUE TITLE
Deactivate User Button

### ISSUE THREAD

**Issue by drjakob** [tactilenews/100eyes#417]
As editor
I want to have a deactivate-user-button
in order to soft-delete users who don't want to get messages anymore.

**Comment by FrauCsu** [comment]
hard-delete?

**Comment by drjakob** [comment]
No, hard-delete Would delete all previous replies. Our archieve.

**Comment by drjakob** [comment]
We‘ll need both. But I think soft-delete is mire urgent
